app/Traits/PreventPastEntryEdits.php

```
@@ -0,0 +1,23 @@
+
+declare(strict_types=1);
+
+namespace App\Traits;
+
+use App\Models\JournalEntry;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Carbon\Carbon;
+
+trait PreventPastEntryEdits
+{
+    private function preventPastEditsAllowed(JournalEntry $entry, int $days = 7): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$entry'. (undefined)

(UnusedFormalParameter)